Update release doc (#67)
apache/skywalking-cli: docs/How-to-release.md

# Apache SkyWalking CLI release guide

This documentation guides the release manager to release the SkyWalking CLI in the Apache Way, and also helps people to check the release for vote.

## Prerequisites

> 1. Clear all milestone issues from [skywalking-cli](https://github.com/apache/skywalking-cli/milestones) and [skywalking](https://github.com/apache/skywalking/milestones), create a new one if needed.

Reviewer: ```suggestion
1. Close(if finished, or move to next milestone otherwise) all issues in the current milestone from [skywalking-cli](https://github.com/apache/skywalking-cli/milestones) and [skywalking](https://github.com/apache/skywalking/milestones), create a new milestone if needed.
```

2. Update [CHANGES.md](../CHANGES.md).

## Add your GPG public key to Apache svn

1. Upload your GPG public key to a public GPG site, such as [MIT's site](http://pgp.mit.edu:11371/). 

1. Log in [id.apache.org](https://id.apache.org/) and submit your key fingerprint.

> 1. Add your GPG public key into [SkyWalking GPG KEYS](https://dist.apache.org/repos/dist/release/skywalking/KEYS) file, **you can do this only if you are a PMC member**.  You can ask a PMC member for help. **DO NOT override the existed `KEYS` file.**

Reviewer: ```suggestion
1. Add your GPG public key into [SkyWalking GPG KEYS](https://dist.apache.org/repos/dist/release/skywalking/KEYS) file, **you can do this only if you are a PMC member**.  You can ask a PMC member for help. **DO NOT override the existed `KEYS` file content, only append your key at the end of the file.**
```

 * https://github.com/apache/skywalking-cli/blob/v$VERSION/docs/How-to-release.md

Voting will start now and will remain open for at least 72 hours, all PMC members are required to give their votes.

[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

Please take part in the vote process.

> Everyone could vote (though may be no-binding), but make sure to check the list [1].

Reviewer 2: `check the list` is only recommended not required. Even PMC members could just choose a subset to check some of them.  
The basic line is making sure the release following ASF requirements and doesn't bring any IP and LICENSE issue.

Author: ok, deleted

   xxx
   xxx
   xxx
   ...
   (list names)
    
   I’ll continue the release process.
   ```

## Publish release

1. Move source codes tar balls and distributions to `https://dist.apache.org/repos/dist/release/skywalking/`, **you can do this only if you are a PMC member**.

    ```shell
    export SVN_EDITOR=vim
    svn mv https://dist.apache.org/repos/dist/dev/skywalking/cli/$VERSION https://dist.apache.org/repos/dist/release/skywalking/cli
    # ....
    # enter your apache password
    # ....
```
    
1. Refer to the previous [PR](https://github.com/apache/skywalking-website/pull/118), update news and links on the website. There are seven files need to modify.

1. Update [Github release page](https://github.com/apache/skywalking-cli/releases), follow the previous convention.

1. Send ANNOUNCE email to `dev@skywalking.apache.org` and `announce@apache.org`, the sender should use his/her Apache email account. You can get the permlink of vote thread at [here](https://lists.apache.org/list.html?dev@skywalking.apache.org).

```text
Subject: [ANNOUNCEMENT] Apache SkyWalking CLI $VERSION Released

Content:

Hi the SkyWalking Community

> On behalf of the SkyWalking CLI Team, I’m glad to announce that SkyWalking CLI $VERSION is now released.

Reviewer 2: ```suggestion
On behalf of the SkyWalking Team, I’m glad to announce that SkyWalking CLI $VERSION is now released.
```

PMC is for the whole project(SkyWalking), SkyWalking CLI is a sub-project.

> SkyWalking CLI: A command line interface for SkyWalking.

Reviewer 2: You should not remove `Apache` before `SkyWalking`. It is not recommended.

Author: reverted the change
